docs/examples/go-dep.ort.yml

```diff
@@ -1,3 +1,3 @@
-excludes:
+excludes: {}
```

Comment:
I guess these fixes were meant to go to a separate commit?

Reply:
No, they were required to make the new tests pass.

Reply:
Yeah, I got that, but I still thought you might have wanted to separate fixing the examples from adding the tests. I'm fine with keeping the changes in one commit now that you've improved the commit message in that regard. However, (very) strictly speaking "were not valid YAML" is not correct for `docs/examples/sbt.ort.yml` which *was* valid YAML but contained an invalid ORT exclude reason.